Open AVAX port 9651 & misc minor documentation updates

ConanMishler opened the pull request:
AVAX seems to sync faster if its P2P port (9651) is open to the outside world. I've tested this change and it doesn't break anything.

Also updated `auto_test/README.md` and the help message given by `cli/projects.py`.

walkjivefly reviewed (changes requested):
Changes look good so far as they go. I pulled them into feature-SYS and all ran OK. 

2 nits:
1. README.md line 51 should say
```
-$ python3 snode_methods http://127.0.0.1:41414 user pass xbridge_methods.json
+$ python3 snode_methods.py http://127.0.0.1:41414 user pass xbridge_methods.json
```
2. There is no auto_test/unit_test.py so lines 8-43 of README.md are redundant/out of sync with reality.

ConanMishler commented:
Good catches, thanks!
1. Changed `snode_methods` to `snode_methods.py`
2. Turns out `auto_test/unit_test.py` was renamed to `snode_auto_test/unit_test.py`. I've updated the README accordingly.
